source/_docs/scripts/conditions.markdown

```diff
@@ -306,58 +306,64 @@ condition:
 
 The sun condition can also test if the sun has already set or risen when a trigger occurs. The `before` and `after` keys can only be set to `sunset` or `sunrise`. They have a corresponding optional offset value (`before_offset`, `after_offset`) that can be added, similar to the [sun trigger][sun_trigger]. When both keys are used, the result is a logical `and` of separate conditions.
 
-Note that if only `before` key is used, the condition will be `true` _from midnight_ until sunrise/sunset. If only `after` key is used, the condition will be `true` from sunset/sunrise _until midnight_. Therefore, to cover time between sunset and sunrise one need to use `after: sunset` and `before: sunrise` as 2 separate conditions and combine them using `or`.
+Note that if only `before` key is used, the condition will be `true` for the time period _from midnight_ until sunrise/sunset. If only `after` key is used, the condition will be `true` from sunset/sunrise _until midnight_. Therefore, to cover time between sunset and sunrise one need to use `after: sunset` and `before: sunrise` as 2 separate conditions and combine them using `or`.
 
 [sun_trigger]: /docs/automation/trigger/#sun-trigger
 
-<div class='note warning'>
-The sunset/sunrise conditions do not work in locations inside the polar circles, and also not in locations with a highly skewed local time zone.
-
-In those cases it is advised to use conditions evaluating the solar elevation instead of the before/after sunset/sunrise conditions.
-</div>
+For a 'when light' condition we can simply use both keys.  This is equivalent to a state condition on `sun.sun` of `above_horizon`.
 
 ```yaml
 condition:
-  condition: sun
-  after: sunset
-  after_offset: "-01:00:00"
+  - condition: sun
+    after: sunrise
+    before: sunset
 ```
 
-This is an example of 1 hour offset after sunset.
+However, we cannot use both keys in a single condition to create an equivalent 'when dark' conditon, as it would always be `false`. The equivalent to a state condition on `sun.sun` of `below_horizon`must use an `or`.
```

Comment:
This is not correct.

```yaml
condition: sun
after: sunrise
before: sunset
```

☝️ That should work according to code. If not, please file a bug report.

Reply:
Your example is for the "when light" situation not the "when dark" situation being explained.  If "when dark" now works without `or`, there is a large text block at the beginning of the section that is no longer necessary.

Reply:
It always worked without the or...

Reply:
The "when dark" situation being reference by this passage is:
```yaml
condition: sun
before: sunrise
after: sunset
```
 The `or` language has been in the docs regarding "when dark" since 2018 in response to [issue #12308](https://github.com/home-assistant/core/issues/12308)